src/data/astarte_message_hub_node.rs

```diff
@@ -354,10 +361,12 @@ mod tests {
 
         let (server_handle, drop_sender, port) = run_local_server(msg_hub).await;
 
+        let (store, tmp_store_path) = create_tmp_store().await;
```

Comment:
Refactor this to remove the dir created before and use the tmp_store_path to write the interface file in

Reply:
Same in the other three places